lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -43,25 +56,38 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy {
     allowOnlyPredefinedValues = true;
 
     @Output()
-    optionsChanged: EventEmitter<string[]> = new EventEmitter();
+    optionsChanged: EventEmitter<AutocompleteOption[]> = new EventEmitter();
+
+    @Output()
+    inputChanged: EventEmitter<string> = new EventEmitter();
 
     readonly separatorKeysCodes = [ENTER] as const;
     formCtrl = new FormControl('');
-    filteredOptions$: Observable<string[]>;
-    selectedOptions: string[] = [];
+    filteredOptions: AutocompleteOption[] = [];
+    selectedOptions: AutocompleteOption[] = [];
+    tooltipShowDelay = 800;
     private onDestroy$ = new Subject<void>();
 
-    constructor() {
-        this.filteredOptions$ = this.formCtrl.valueChanges.pipe(
-            startWith(null),
-            map((value: string | null) => (value ? this.filter(value) : []))
-        );
-    }
-
     ngOnInit() {
+        this.formCtrl.valueChanges
+            .pipe(
+                startWith(''),
+                debounce((value: string) => (value ? timer(300) : EMPTY)),
```

Comment:
can I ask why we need debounce here? I guess we don't do any backend calls etc. so why exactly is that needed?